game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/apeatroll/daga.plugin.kts

```diff
@@ -0,0 +1,87 @@
+package gg.rsmod.plugins.content.areas.apeatroll
+
+import gg.rsmod.plugins.content.mechanics.shops.CoinCurrency
+
+/**
+ * @author Alycia <https://github.com/alycii>
```

Comment:
That's nice of you to give credit to Alycia, but perhaps she didn't work on this?